packages/tasks/src/model-libraries.ts

```diff
@@ -229,6 +229,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Unbabel/COMET/",
 		countDownloads: `path:"hparams.yaml"`,
 	},
+	"chronos-forecasting": {
```

Comment:
can you move it up in the mapping (just after `chat_tts`) to keep alphabetical order? Thanks!